src/System.Azure.Experimental/System/Azure/StorageAccessSignature.cs

```diff
@@ -6,79 +6,56 @@
 using System.Buffers;
 using System.Buffers.Text;
 using System.Buffers.Cryptography;
-using System.Text;
+using System.Text.Utf8;
 
 namespace System.Azure.Authentication
 {
     public static class StorageAccessSignature
     {
+        static Utf8String s_emptyHeaders = (Utf8String)"\n\n\n\n\n\n\n\n\n\n\nx-ms-date:"; // this wont be beeded once we have UTF8 literals
+
         public static bool TryWrite(Span<byte> output, Sha256 hash, string verb, string canonicalizedResource, DateTime utc, out int bytesWritten)
         {
-            int written, consumed;
-            bytesWritten = 0;
-
-            if (verb.Equals("GET", StringComparison.Ordinal))
+            try
             {
-                if (output.Length < 3)
-                {
-                    bytesWritten = 0;
-                    return false;
-                }
-                s_GET.CopyTo(output);
-                bytesWritten += s_GET.Length;
+                var writer = new SpanWriter(output);
+                writer.WriteLine(verb);
+                writer.Write("\n\n\n\n\n\n\n\n\n\n\nx-ms-date:");
```

Comment:
Use `s_emptyHeaders`?

Reply:
s_emptyHeaders is Utf8String. I wanted this one to be UTF16